src/Symfony/Component/Notifier/Bridge/Telnyx/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/telnyx-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony Telnyx Notifier Bridge",
+    "keywords": ["sms", "telnyx", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.0",
```

Comment:
I would use `4.4` as min version rather than `4.3` as 4.3 is already EOL

Reply:
Fixed